data_structures/queue_with_stacks.py

```diff
@@ -0,0 +1,34 @@
+from stack.stack import Stack
+
+
+class Queue_with_stacks:
+
+    def __init__(self, iterable=[]):
+        self.stack_back = Stack(iterable)
+        self.stack_front = Stack()
+
+    def __topval__(self):
```

Comment:
Private methods only have a single underscore before them, double underscores are for magic methods only by convention. Thus this should be `_topval(self)` instead.